<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoftRockOptions</class>
 <widget class="QWidget" name="SoftRockOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>3</y>
     <width>104</width>
     <height>129</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>BandPass Filters</string>
   </property>
   <widget class="QRadioButton" name="automaticButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Automatic</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="filter0Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>38</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter 0</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="filter1Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>58</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="filter2Button">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter 2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="filter3Button">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>102</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>18</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter 3</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="versionLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>6</y>
     <width>156</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Version:</string>
   </property>
  </widget>
  <widget class="QLabel" name="si570Label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>48</y>
     <width>148</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Si570:</string>
   </property>
  </widget>
  <widget class="QLabel" name="serialLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>27</y>
     <width>107</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Serial #:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="serialBox">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>24</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>28</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Serial #</string>
   </property>
  </widget>
  <widget class="QLabel" name="fifiVersionLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>92</y>
     <width>156</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FiFi Version:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fifiHelp">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>73</y>
     <width>352</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;FiFi: Use rockprog for options not exposed in this dialog&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
